asset-schemas/ontology_info_schema.json

```diff
@@ -65,7 +65,6 @@
         }
       },
       "required": [
-        "version",
```

Comment:
sorry if i've missed existing discussion of this (link me if so)--but have we already talked w/ Lattice and Brian on the implications of cellosaurus not having a version? Its against the purpose of COG to not be able to trace what ontology is being used in each release; what terms are considered valid or their metadata can change between releases w/o a way to track the differences. 

if they're not tracking versions, should we version it ourselves? maybe by date? is there any way for us to determine that changes have occurred since the last time we pulled? maybe if there's a diff in the build we track it as a new version?

Reply:
Cellosaurus uses an embedded date which can be introspected. 

The current release is Release 53 of August 2025 which is embedded in the obo:

`data-version: 53.0`

Reply:
would we prefer the data or the version or a combination?